packages/eslint-config-turbo/src/flat/index.ts

```diff
@@ -5,7 +5,7 @@ import plugin from "eslint-plugin-turbo";
 export default [
   {
     plugins: {
-      turbo: plugin,
+      turbo: [plugin],
```

Comment:
Unneeded with https://github.com/vercel/turborepo/pull/10157
```suggestion
      turbo: plugin,
```

Reply:
Ope, yup, forgot to drop that change while I was changing branches. Good catch.